LoopKitUI/Views/DismissibleKeyboardTextField.swift

```diff
@@ -16,30 +16,28 @@ public struct DismissibleKeyboardTextField: UIViewRepresentable {
     var textColor: UIColor
     var textAlignment: NSTextAlignment
     var keyboardType: UIKeyboardType
+    var didBecomeFirstResponder: Bool
```

Comment:
[nit] I'm probably misunderstanding the intention, but this seems like it should be named "shouldBecomeFirstResponder". After all, it doesn't actually change if it did become the first responder.

Reply:
You're right; this one should be `shouldBecomeFirstResponder`, and the one in the Coordinator should stay `didBecomeFirstResponder`.